src/general/Species.cs

```diff
@@ -46,6 +46,8 @@ protected Species(uint id, string genus, string epithet)
 
     public Color Colour { get; set; } = new(1, 1, 1);
 
+    public Color ColourGUI => new(Colour.r, Colour.g, Colour.b, 1);
```

Comment:
I'd personally want to see this named `GUIColour` so that this reads naturally (while I do get that having a suffix can be better for discoverability.

Also this reads the Colour property 3 times so this results in 3 method calls on each read...
So I'd like to see this written like:
```
get
{
    var colour = Colour;
    return new Color(...);
}
```

Reply:
Oh and one more thing, there should be an xml comment on this property explaining why it exists (and is different from Colour) and when it should be used.